Makefile

```diff
@@ -366,6 +387,10 @@ loop: mod core
 	@$(CC) -c -o $(PREFIX)/loop_ping.o test/loop_ping.c $(CFLAGS) -ggdb
 
 units: mod core lib $(UNITS_OBJ) $(MOD_OBJ)
+	@if [ $(UNITS) -eq 0 ]; then \
```

Comment:
@frederikvs can't we just define the UNITS=1 variable inside this target? Since you're always setting it, I don't see any reason why it should be passed as a parameter.